src/index.html

```diff
@@ -10,8 +10,164 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>HTML Form</h1>
+  <body class="body">
+    <form
+      method="post"
+      action="https://mate-academy-form-lesson.herokuapp.com/   create-application"
+      class="form">
+
+      <fieldset  class="field-form" name="personalInformation">
+        <legend>Personal information</legend>
+        <label class="input-form">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+        <label class="input-form">
+          Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+        <label class="input-form">
+         How old are You?:
+          <input
```

Comment:
please check the spacing
```suggestion
        <label class="input-form">
          How old are You?:
          <input
```

Reply:
not fixed